.gitignore

```diff
@@ -1 +1,5 @@
-node_modules
\ No newline at end of file
+node_modules
+yarn-error.log
+lerna-debug.log
+yarn.lock
+__tests__/
```

Comment:
Can we add an empty line at the end of the line?

Reply:
Done